packages/components/src/counter/src/Counter.css

```diff
@@ -1,26 +1,27 @@
 .o-ui-counter {
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
```

Comment:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/8532e3a8-6449-417a-af85-cf83d800f651)
As the font size changes drastically have you validated that it looks good when embedded in other components?

Reply:
I don't think the font size was changed, just the font weight. When looking at the chromatic tests, it looked fine in the other components.

Reply:
https://orbit.sharegate.design/ it used to be 14px now it's 16